packages/core/src/utils/CoreHelperUtil.ts

```diff
@@ -129,15 +129,16 @@ export const CoreHelperUtil = {
   formatBalance(balance: string | undefined, symbol: string | undefined) {
```

Comment:
are we also removing decimals from the account view balance? If not, we should receive the decimals as params

Reply:
I'm not very sure ,but soon we will all find out

Reply:
We are not removing decimals, we only show 3 now